lib/storage/metadata/mongoclient/MongoClientInterface.js

```diff
@@ -1054,6 +1054,7 @@ class MongoClientInterface {
                             isVersioned:
                                 !!bucketInfo.getVersioningConfiguration(),
                             ownerCanonicalId: bucketInfo.getOwner(),
+                            ingestion: bucketInfo.isIngestionEnabled(),
```

Comment:
For the management layer's purposes, this should not change when ingestion is paused, what we need here is the nature of the bucket (ingest vs normal), not the current state.

Reply:
I've added a new function that would check if the bucket is/is not an ingestion bucket, and replaced it here :)